client/me/get-apps/desktop-download-card.jsx

```diff
@@ -0,0 +1,171 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import { recordTracksEvent } from 'state/analytics/actions';
```

Comment:
With redux actions, we normally can't use them directly. This should be passed through connect in the `mapDispatchToProps` argument.
https://github.com/reactjs/react-redux/blob/master/docs/api.md#arguments
eg:
```jsx
const mapDispatchToProps = {
    trackWindowsDownload: partial( recordTracksEvent, 'calypso_app_download_windows_download' ),
     trackMacDownload: partial( recordTracksEvent, 'calypso_app_download_mac_download' ),
     // .. etc
};
export default connect(
    null,
    mapDispatchToProps
)( localize( DesktopDownloadCard ) );
```
Let's also avoid using translate directly from i18n. It's minor, but we can miss some updates for certain lifecycle events.